src/ImageSharp/Compression/Zlib/Adler32.cs

```diff
@@ -83,18 +90,16 @@ private static unsafe uint CalculateSse(uint adler, ReadOnlySpan<byte> buffer)
             uint s2 = (adler >> 16) & 0xFFFF;
 
             // Process the data in blocks.
-            const int BLOCK_SIZE = 1 << 5;
-
             uint length = (uint)buffer.Length;
-            uint blocks = length / BLOCK_SIZE;
-            length -= blocks * BLOCK_SIZE;
+            uint blocks = length / BlockSize;
+            length -= blocks * BlockSize;
 
             int index = 0;
             fixed (byte* bufferPtr = buffer)
             {
                 fixed (byte* tapPtr = Tap1Tap2)
```

Comment:
```suggestion
            fixed (byte* bufferPtr = &MemoryMarshal.GetReference(buffer))
            {
                fixed (byte* tapPtr = &MemoryMarshal.GetReference(Tap1Tap2))
```
For `buffer` there's the empty-check done before, so no need to repeat it (implicitely by GetPinnableReference which C# generates for the fixed statement here).
For `Tap1Tag2` it's a "constant" we know for sure it's not-empty.

Note: also add the namespace needed for _MemoryMarshal_.